File: XBOOK/XBOOK.Web/ClientApp/src/app/modules/_shared/models/supplier/supplier-view.model.ts
import { BaseModel } from '@shared/model/base.model';
export class SupplierView extends BaseModel {
    supplierName: string;
    address: string;
    taxCode: number;
    tag: number;
    contactName: string;
    email: string;
    note: string;
    supplierID: number;
    bankAccount: string;
}
export class SupplierViewModel {
    supplierID: number;
    supplierName: string;
    address: string;
    note: string;
    outstanding: number;
    overdue: number;
    bankAccount: string;
}

File: XBOOK/XBOOK.Dapper/ViewModels/SupplierViewModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace XBOOK.Dapper.ViewModels
{
    public class SupplierViewModel
    {
        public int SupplierID { get; set; }
        public string SupplierName { get; set; }
        public string Address { get; set; }
        public string Note { get; set; }
        public decimal Outstanding { get; set; }
        public decimal Overdue { get; set; }
        public string TaxCode { get; set; }
        public string Tag { get; set; }
        public string ContactName { get; set; }
        public string Email { get; set; }
        public string BankAccount { get; set; }
    }
}
